Sources/ISDBTibs/TibsToolchain.swift

```diff
@@ -55,7 +55,7 @@ public final class TibsToolchain {
   }()
 
   public private(set) lazy var clangHasIndexSupport: Bool = {
-    clangVersionOutput.starts(with: "Apple") || clangVersionOutput.contains("swift-clang")
+    clangVersionOutput.starts(with: "Apple") || clangVersionOutput.contains("apple/llvm-project")
```

Comment:
The clang from Swift's forked repo doesn't put out either of these on Android. I checked on linux, and neither the latest 5.1.3 release for Ubuntu nor a snapshot release from master say "swift-clang", so I'm not sure how the CI doesn't show this warning I got on Android before I modified this:
```
Test Case 'IndexTests.testMixedLangTarget' started at 2019-12-26 02:07:33.359
warning: skipping test because '/data/data/com.termux/files/home/src/swift/build/Ninja-Release/toolchain-android-aarch64/usr/bin/clang' does not have indexstore support; use swift-clang
Test Case 'IndexTests.testMixedLangTarget' passed (0.153 seconds)
```
The 5.1.3 release had neither, while the snapshot release had this new string, probably ever since the switch to the LLVM monorepo. Maybe the CI is picking up some other clang installed in the system?

I'm unsure if the 5.2 release will have this string or not, so this change is tentative for at least the master branch.

Reply:
Good catch! The test is being skipped on our Linux CI right now for the same reason.  We should probably make it mandatory at least on CI, but that's a separate issue.  I believe we will see

5.1 and earlier: "swift-clang"
5.2 and later: "apple/llvm-project"

We should change this code to check for both cases, at least for now.